time/index.html

```diff
@@ -73,10 +73,12 @@
 </section>
 
 <section id="sotd">
-<p><strong>For OGC</strong> </p>
-<p>This is a Public Draft of a document prepared by the Spatial Data on the Web Working Group (<a href="http://www.opengeospatial.org/projects/groups/sdwwg">SDWWG</a>)
-— a joint W3C-OGC project (see <a href="https://www.w3.org/2015/spatial/charter">charter</a>). The document is
-prepared following W3C conventions. The document is released at this time to solicit public comment.</p>
+<p><strong>For OGC -</strong> This is a Public Draft of a document prepared by the Spatial Data on the Web Working Group 
+	(<a href="http://www.opengeospatial.org/projects/groups/sdwwg">SDWWG</a>)
+— a joint W3C-OGC project (see <a href="https://www.w3.org/2015/spatial/charter">charter</a>). 
+The document is prepared following W3C conventions. </p>
+<p>The group expects to request transition to Proposed Recommendation once it has reviewed datasets that have adopted the ontology and completed the implementation report</p>
+<p class="issue">New classes and properties are introduced in this revision of OWL-Time. The new elements primarily relate to relaxing the limitation that time position uses the Gregorian Calendar, and are placed in a logical hierarchy with the original elements. While there is naturally less implementation evidence for these than the elements that were introduced in the 2006 version, these elements are essential to satisfying key requirements in the revision. However, a small number of other new elements have been introduced which have been less strongly motivated and tested, so these are formally <i>features at risk</i> pending the accumulation of evidence of implementation. The features at risk are clearly indicated in this document and in the RDF representation of the ontology. </p>
```

Comment:
Actually, given the restricted number of features at risk, I would list them explicitly here. Italics are easy to miss.